src/import.cmd.js

```diff
@@ -65,6 +80,35 @@ export default class ImportCommand extends AbstractCommand {
     this.emit('stopped', this);
   }
 
+  async setupImporterUI() {
+    const importerFolder = path.join(this.directory, this._importerSubPath);
+    await fse.ensureDir(importerFolder);
+    const uiProjectName = path.basename(this._uiRepo, '.git');
+    const uiFolder = path.join(importerFolder, uiProjectName);
+    const exists = await fse.pathExists(uiFolder);
+    if (!exists) {
+      this.log.info('Helix Importer UI needs to be installed.');
+      this.log.info(`Cloning ${IMPORTER_UI_REPO} in ${importerFolder}.`);
+      // clone the ui project
+      await git.clone({
+        fs: fse,
+        http,
+        dir: uiFolder,
+        url: IMPORTER_UI_REPO,
+        ref: 'main',
+        depth: 1,
+        singleBranch: true,
+      });
```

Comment:
is this fast? otherwise use just use git, like we did here:
https://github.com/adobe/helix-cli/blob/192f8410af0aff35eb2938080f77fa01cb0a0bb3/src/helix-pages.js#L88-L95

Reply:
Yes, it is fast. But this I already use `shelljs`, I can remove the `isomorphic-git` dependency. Would do the same anyway.